lib/sycamore/sycamore/query/execution/sycamore_executor.py

```diff
@@ -53,7 +54,6 @@ class SycamoreExecutor:
     def __init__(
         self,
         context: Context,
-        os_client_args: Any,
         s3_cache_path: Optional[str] = None,
```

Comment:
If we're gonna move the os_client_args out of here, it seems to me that s3_cache_path should be part of the context as well.

Reply:
oh yeah this was a miss